setup.py

```diff
@@ -12,21 +12,23 @@
 
 # https://packaging.python.org/guides/single-sourcing-package-version/
 # http://blog.ionelmc.ro/2014/05/25/python-packaging/
-
 PATH_ROOT = os.path.dirname(__file__)
 builtins.__LIGHTNING_SETUP__ = True
 
 import pytorch_lightning  # noqa: E402
 
 
-def load_requirements(path_dir=PATH_ROOT, comment_char='#'):
-    with open(os.path.join(path_dir, 'requirements', 'base.txt'), 'r') as file:
+def load_requirements(path_dir=PATH_ROOT, file_name='base.txt', comment_char='#'):
+    with open(os.path.join(path_dir, 'requirements', file_name), 'r') as file:
         lines = [ln.strip() for ln in file.readlines()]
     reqs = []
     for ln in lines:
         # filer all comments
         if comment_char in ln:
-            ln = ln[:ln.index(comment_char)]
+            ln = ln[:ln.index(comment_char)].strip()
+        # Make slight syntax alteration to git dependency for PL's sphinx theme
+        if ln.startswith('git') and file_name == 'docs.txt':
```

Comment:
I would rather define some lookup table as there may be in future some other in other files

Reply:
Can we cross this bridge when we come to it? Don't want to overcomplicate